### DISCUSSION TITLE
Error updating feeds

### DISCUSSION THREAD

**Discussion by wellenreiten54** [immauss/openvas#301] [Q&A]
Currently, it is not possible to update the feeds in my installation. The following error message appears: 

Running as root. Switching to user 'gvm' and group 'gvm'.
Trying to acquire lock on /var/lib/gvm/feed-update.lock
Acquired lock on /var/lib/gvm/feed-update.lock
⠇ Downloading CERT-Bund data from rsync://feed.community.greenbone.net/community/vulnerability-feed/22.04/cert-data/ to /var/lib/gvm/cert-data
rsync: server sent "<!DOCTYPE html>" rather than greeting
rsync error: error starting client-server protocol (code 5) at main.c(1863) [Receiver=3.2.7]

Releasing lock on /var/lib/gvm/feed-update.lock

What could be the problem?

**Comment by immauss** [comment]
This could be your connectivity, or it could have been a transient error with the Greenbone Feed servers. Give it another try and if you are still having issues, please let me know. 

-Scott

**Reply by wellenreiten54** [reply]
I have tried it several times and it always failed. For some reason, it worked this time. Thank you for your support.
